AndrewTehan/hometask-3/cli.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+ENTER_MESSAGE = 'Enter city name: '
+ERROR_MESSAGE = 'City name can not be blank!'
+EXIT_MESSAGE = 'exit!'
+
+# this class contains describes cli for rainfall stats
+class Cli
+  def run_cli
+    loop do
+      input = wait_for_input
+
+      exit if input == EXIT_MESSAGE
+
+      if input.empty?
+        show_error
+      else
+        show_result(input)
+      end
```

Comment:
Why don't you use ternary operator `input.empty? ? show_error : show_result(input)`?

it looks lighter than if-else construction